module-apps/application-notes/windows/NotesOptions.cpp

```diff
@@ -12,6 +12,7 @@
 #include <i18n/i18n.hpp>
 
 #include <module-gui/gui/widgets/text/Text.hpp>
+#include <AppWindowConsts.hpp>
```

Comment:
Constants ? We similar have TextConstants

Reply:
applied constants name